torch/testing/_internal/common_methods_invocations.py

```diff
@@ -1865,12 +1865,29 @@ def sample_inputs_amax_amin(op_info, device, dtype, requires_grad, **kwargs):
     # Ordered as (shape, positional args, kwargs)
     test_cases: Tuple[tuple, tuple, dict] = (  # type: ignore[assignment]
         ((S, S, S), (), {}),
-        ((S, S, S), (1,), {}),
-        ((S, S, S), ((1, 2,),), {}),
-        ((S, S, S), (1,), {'keepdim': True}),
-        ((), (0,), {}),
+        ((S, S, S), (), {'dim': 1}),
+        ((S, S, S), (), {'dim': (1, 2,)}),
+        ((S, S, S), (), {'dim': 1, 'keepdim': True}),
+        ((), (), {'dim': 0}),
         ((), (), {}),
-        ((), (0,), {'keepdim': True}),
+        ((), (), {'dim': 0, 'keepdim': True}),
+    )
+    return tuple(SampleInput((make_tensor(size, device, dtype,
```

Comment:
Style nit: prefer starting with the for loop for readability (vs putting it at the end of the statement, which is yoda-like)